build.py

```diff
@@ -13,5 +13,9 @@
     os.system("conan package .. --build_folder=../")
     print(os.listdir("."))
     print(os.listdir("./lib"))
+    os.system("cd .. && conan export user/channel") # This will improve next 0.25 with an --cwd=.. arg
+    os.system("conan package_files Test/0.1@user/channel -f")  # -f in case it exists, overwrite
+    os.system("conan search Test/0.1@user/channel") # To show the package binary
+    # os.system("conan upload Test/0.1@user/channel --all -r=")
```

Comment:
For consumption: my recommendation when it gets serious and into production: be very consistent with settings/options. Much better to use profiles:

```
$ conan install -pr=my_vs14_release_profile # poor name, use your own
$ ...
$ conan package_files Pkg/version@user/channel -pr=my_vs14_release_profile
# to reuse it:
$ conan install -pr=my_vs14_release_profile
```